tron/serialize/runstate/statemanager.py

```diff
@@ -14,6 +15,8 @@
 from tron.serialize.runstate.yamlstore import YamlStateStore
 from tron.utils import observer
 
+# import threading
```

Comment:
if we go with this, reminder to remove the commented code before final review :)

Reply:
yup, was leaving this for potential "threading.Lock()" usage